### DISCUSSION TITLE
Authntication

### DISCUSSION THREAD

**Discussion by Swapnil-1991** [community/community#29166] [Other Feature Feedback, Questions, & Ideas]
Support for authentication was removed from 13 August 2021. How to fix this

**Comment by airtower-luna** [comment]
You're much more likely to get a helpful response if you provide details on what you're doing (exact command, full output).

That message looks vaguely similar to the one you get if you try to do `git push` or similar with your password. You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and use that instead of your password when Git asks.
